src/components/CallView/Grid/Grid.vue

```diff
@@ -968,6 +969,11 @@ export default {
 	position: relative;
 	bottom: 0;
 	inset-inline-start: 0;
+
+	&:has(.stripe-wrapper) {
+		bottom: var(--grid-gap);
```

Comment:
Some spacing would be nice to have here:

<img width="772" height="104" alt="Image" src="https://github.com/user-attachments/assets/e9d8085a-844d-443b-b271-f88ee2666cf3" />